src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-lt.php

```diff
@@ -292,6 +292,81 @@
     '🏴󠁧󠁢󠁳󠁣󠁴󠁿' => 'vėliava: Škotija',
     '🏴󠁧󠁢󠁷󠁬󠁳󠁿' => 'vėliava: Velsas',
     '🧑‍🧑‍🧒‍🧒' => 'šeima: suaugusysis, suaugusysis, vaikas, vaikas',
+    '🚶🏻‍♀‍➡' => 'einanti moteris atsisukęs į dešinę',
+    '🚶🏼‍♀‍➡' => 'einanti moteris atsisukęs į dešinę',
+    '🚶🏽‍♀‍➡' => 'einanti moteris atsisukęs į dešinę',
+    '🚶🏾‍♀‍➡' => 'einanti moteris atsisukęs į dešinę',
+    '🚶🏿‍♀‍➡' => 'einanti moteris atsisukęs į dešinę',
```

Comment:
how there are generated? `atsisukęs` should be used for with `vyras`, but for `moteris` - `atsisukus`

Reply:
This is generated automatically, based on data published by ICU team.
See
* https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Emoji/Resources/bin/build.php
* https://github.com/unicode-org/cldr/blob/main/common/annotationsDerived/lt.xml#L2703